process/block/preprocess/transactions.go

```diff
@@ -70,6 +70,8 @@ func NewTransactionPreprocessor(
 	pubkeyConverter core.PubkeyConverter,
 	blockSizeComputation BlockSizeComputationHandler,
 	balanceComputation BalanceComputationHandler,
+	epochNotifier process.EpochNotifier,
```

Comment:
Should add a TODO to change this long list of arguments to a single Arg struct.

Reply:
Done